rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
+  "optIn": [{ "click": ".sibbo-layout [data-accept-all]" }],
+  "optOut": [
+    {
+      "if": { 
+        "exists": "[data-view=purposes] .sibbo-panel__main__footer__actions [data-save-and-exit]"
+      },
+      "then": [
+        { 
+          "click": ".sibbo-panel__aside__buttons a[data-nav=\"purposes\"]"
+        },
+        { 
+          "click": ".sibbo-panel__main__header__actions a[data-focusable=\"reject-all\"]" 
+        },
```

Comment:
these steps, and two last steps are the same in both branches. Can we move them out of the "if" block?
The if condition can probably be based on existensce of `.sibbo-panel__main__footer__actions a[data-focusable=\"next\"]:not(.sibbo-cmp-button--disabled)`, in this case. What do you think?

Reply:
Regarding the following selector `.sibbo-panel__main__footer__actions a[data-focusable=\"next\"]:not(.sibbo-cmp-button--disabled)`, it can't be used as the "if condition" since it appears on both models of banners (on the first settings page).

From what I saw, Sibbo has reused the 2-page banner's HTML to build its 1-page banner model. The former being the main pattern their clients are using.

Therefore, it seems they only added one selector to the 1-page banner model to distinguish between the two models.